src/Propel/Generator/Reverse/PgsqlSchemaParser.php

```diff
@@ -325,6 +325,34 @@ protected function addColumns(Table $table, $oid)
         }
     }
 
+    /**
+     * @param string $default
+     *
+     * @return bool
+     */
+    protected function isColumnDefaultExpression($default)
+    {
+        $containsFunctionCall = "'" !== substr($default, 0, 1) && strpos($default, '(');
+
+        if ($containsFunctionCall) {
+            return true;
+        }
+
+        $defaultColumnValueExpressions = [
+            'CURRENT_DATE' => 'CURRENT_DATE',
+            'CURRENT_TIME' => 'CURRENT_TIME',
+            'CURRENT_TIMESTAMP' => 'CURRENT_TIMESTAMP',
+            'LOCALTIME' => 'LOCALTIME',
+            'LOCALTIMESTAMP' => 'LOCALTIMESTAMP'
```

Comment:
```suggestion
            'LOCALTIME' => 'LOCALTIME',
            'LOCALTIMESTAMP' => 'LOCALTIMESTAMP'
```

The others require pgsql-specific tests as they don't work on other db engines.